common/config/rush/pnpm-config.json

```diff
@@ -17,7 +17,8 @@
     }
   },
   "globalOverrides": {
-    "dottie": "^2.0.4"
+    "dottie": "^2.0.4",
+    "@babel/traverse": "7.23.2"
```

Comment:
why didnt `rush update --full` work?

Reply:
Rush update --full caused an error during build. I believe it was related to web pack. This is one of the failed builds
https://dev.azure.com/bentleycs/iModelTechnologies/_build/results?buildId=2477530&view=logs&j=9f3a3068-7880-5994-433d-b92c1f9d379b&t=86ae3434-ba71-59ee-b475-59e58c39d505

Reply:
why not address the failures from this?
the next person who needs to update the lockfile for a cve will hit this issue

Reply:
there are 3 errors,
the core-webpack-tools one should be trivial to fix
the presentation-otel one shouldnt be too bad to fix
the core-react might be a little painful, but @raplemie  or @GerardasB should be able to help with that